src/components/login/actions.js

```diff
@@ -0,0 +1,41 @@
+import { LOGIN_USER, AUTHENTICATE_USER } from './actionTypes';
+
+/**
+ * Triggers request to authenticate user and log them in using firebase
+ *
+ * @function
+ * @return {Object} The {@link actionTypes.LOGIN_USER LOGIN_USER} action.
+ */
+const loginUser = password => {
+    // convert the password to string for easy slicing and nomenclature
+    const stringPassword = String(password);
```

Comment:
why are you doing this?

Reply:
i have renamed the variable, but it was a phone number, entered as an integer. i needed to convert it to string so i can append "+234" in front of it